userdocs/src/community.md

```diff
@@ -7,20 +12,60 @@
 
 ## Release Cadence
 
-Starting with 0.2.0 onwards, minor releases of `eksctl` should be expected every two weeks and patch releases will be made available as needed.
+Minor releases of `eksctl` are loosely scheduled for weekly on Fridays. Patch
+releases will be made available as needed.
+
+One or more release candidate(s) (RC) builds will be made available prior to
+each minor release. RC builds are intended only for testing purposes.
+
+## 2021 Roadmap
+
+The following are the features/epics we will focus on and hope to ship this year.
+We will take their completion as a marker for graduation to v1.
+General maintenance of `eksctl` is still implied alongside this work,
+but all subsequent features which are suggested during the year will be weighed
+in relation to the core targets.
+
+Progress on the roadmap can be tracked [here](https://github.com/weaveworks/eksctl/projects/2).
```

Comment:
shall we add the issues to the project before updating the site?

Reply:
i don't think one blocks the other, but is on my list 👍